api_service.py

```
@@ -1542,14 +1545,112 @@ async def trigger_monthly_summary(x_api_key: Optional[str] = Header(None)):
-    return OAuthStateRepository()
+    # Import api_service to allow mocking in tests
+    import api_service as self_module
+    return self_module.oauth_state_repo
+
+
+@app.post("/api/auth/gmail/init", response_model=OAuthStateInitResponse, tags=["oauth"])
+async def oauth_state_init(
+    request_body: OAuthStateInitRequest,
+    http_request: Request,
+    x_api_key: Optional[str] = Header(None),
+    state_repo: OAuthStateRepository = Depends(get_oauth_state_repository),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)